src/raftstore/store/worker/region.rs

```diff
@@ -333,6 +339,35 @@ impl SnapContext {
         Ok(())
     }
 
+    // check the number of files at level 0 to avoid write stall after ingesting sst,
+    // return indicate whether ingest will cause write stall or not.
+    fn ingest_maybe_stall(&self) -> bool {
+        for cf in SNAPSHOT_CFS {
+            // no need to check lock cf
+            if plain_file_used(cf) {
+                continue;
+            }
+
+            let handle = match rocksdb::get_cf_handle(&self.engines.kv, cf) {
```

Comment:
Does it okay to apply snapshots if it failed to get cf handle? @huachaohuang

Reply:
You can just `unwrap` here, TiKV never creates/drops column families dynamically, so the cf must exist unless something is terribly wrong.